cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/TableSegmentRecoveryCommand.java

```diff
@@ -206,8 +206,8 @@ private int scanAllEntriesInTableSegmentChunks(ByteArraySegment byteArraySegment
                         .getReader().readNBytes(header.getValueLength());
                 // Add the operation to the list of operations to replay later on (PUT or DELETE).
                 tableSegmentOperations.add(valueBytes.length == 0 ?
-                        new DeleteOperation(TableSegmentEntry.versioned(keyBytes, valueBytes, header.getEntryVersion())) :
-                        new PutOperation(TableSegmentEntry.versioned(keyBytes, valueBytes, header.getEntryVersion())));
+                        new DeleteOperation(TableSegmentEntry.unversioned(keyBytes, valueBytes)) :
```

Comment:
Could it happen to parse a DELETE operation for a KEY that does not exist? Are handling that case if it can happen?

Reply:
Added a 'DELETE' operation for a KEY that does not exist in the table-segment recovery command for manual testing and the recovery completed without any error.